bin/gce-pre-align.sh

```diff
@@ -80,10 +78,10 @@ then
 
     if [[ $CONTAINER_IS_RUNNING == 0 && $EXIT_STATUS == 0 ]]
     then
-      OUTPUT_DIR=$2
+      OUTPUT_DIR=$2"/"
       echo "[$(date)] Uploading $OUTPUT_FILE"
       START_TIME=$(date +%s)
-      gcloud compute ssh --zone $MACHINE_ZONE $MACHINE_NAME -- gsutil -o GSUtil:parallel_composite_upload_threshold=150M cp /home/alignment/*.fastq.gz $OUTPUT_DIR
+      gcloud compute ssh --zone $MACHINE_ZONE $MACHINE_NAME -- gsutil -o GSUtil:parallel_composite_upload_threshold=150M cp /home/alignment/*.fastq.gz $OUTPUT_DIR && gcloud compute ssh --zone $MACHINE_ZONE $MACHINE_NAME -- gsutil -o GSUtil:parallel_composite_upload_threshold=150M cp ${OUT_BASE}.list $OUTPUT_DIR 
```

Comment:
There is no need to chain these commands with `&&` if you just used an rsync operation. I want to keep any logs and list files generated anyways. Not to mention I've been bitten by `gsutil cp` deleting data.